src/Propel/Runtime/ActiveQuery/Criterion/AbstractInnerQueryCriterion.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+/**
+ * MIT License. This file is part of the Propel package.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Propel\Runtime\ActiveQuery\Criterion;
+
+use Propel\Runtime\ActiveQuery\Criteria;
+use Propel\Runtime\ActiveQuery\ModelCriteria;
+use Propel\Runtime\ActiveQuery\ModelJoin;
+use Propel\Runtime\Exception\PropelException;
+use Propel\Runtime\Map\RelationMap;
+
+/**
+ * Abstract Criterion for nested filter expression, that bind an inner query
+ * with an operator like IN, EXISTS
+ *
+ * @phpstan-consistent-constructor
+ */
+abstract class AbstractInnerQueryCriterion extends AbstractCriterion
+{
+ /**
+  * @var string|null Left side of the operator, can be empty.
+  */
```

Comment:
we might have some cs issues here

Reply:
What do you mean? I don't understand

Reply:
The indentation seems off

Reply:
Right, good catch! My editor does that sometimes. Weird that the sniffer doesn't fix it.